web/src/app/components/share-list/share-list.component.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.role-select {
```

Comment:
Could you kindly update the styling and components used to be closer to the designs?

- [ ] Rounded chip instead of button styling for "Add" button
- [ ] Color of horizontal lines between users
- [ ] Vertical alignment of email addresses
- [ ] Role dropdown styling - font size, color, remove bottom border / underline

![image](https://github.com/google/ground-platform/assets/228050/5d5ea6d1-b389-4771-806e-7461b2017fdc)